test/demos/forwarddiffzero.jl

```diff
@@ -0,0 +1,76 @@
+"The simplest viable forward mode a AD, only supports `Float64`"
+module ForwardDiffZero
```

Comment:
Is there anything that would be trickier for a ReverseDiff-like package using this approach? i.e. is there anything to be learned from having a `ReverseDiffZero` demo?

Reply:
Yeah, unclear.
We might need to make one to find out.

Reply:
I suspect there might be something to learn from ADs that are willing to deal with different sets of types.
E.g. this one only accepts AD on Floats,
but maybe your AD does Floats + Matrixes? (like Tracker)

Reply:
Done